server/controllers/documents/upload/helpers/sendFileSizeMetric.ts

```diff
@@ -0,0 +1,21 @@
+import promClient from 'prom-client'
+import { bytesToMB } from './index'
+import logger from '../../../../../logger'
+
+export const fileUploadHistogram = new promClient.Histogram({
+  name: 'upload_file_sizes',
+  help: 'sizes in MB of uploaded files',
+  buckets: promClient.linearBuckets(0, 1, 100),
```

Comment:
These buckets seem a little granular...  That's basically saying, create 100 buckets, starting at 0mb with 1mb in separation.  This is very high cardinality in prometheus and may cause performance/storage issues.

I'd maybe go with (0, 5, 20)